src/less/textbox/base/textbox.less

```diff
@@ -19,7 +19,7 @@ span.textbox {
         .textbox__icon {
```

Comment:
Sorry @raveenakothapally, one last thing. Can you try @agliga's suggestion of dropping `.textbox__icon` class entirely in favor of `.textbox svg.icon`?

Reply:
Please let me know if the changes look good.